_data/people.yaml

```
@@ -10,6 +10,19 @@
   masters:
   bachelors: University of Washington
   active: false
+  
+  - name: Timothy Akintilo
```

YAML is whitespace sensitive, so just need to tuck this line back to get things to build.
```suggestion
- name: Timothy Akintilo
```